servers/src/main/java/tachyon/worker/block/evictor/LFUEvictor.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker.block.evictor;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Optional;
+
+import tachyon.Constants;
+import tachyon.Pair;
+import tachyon.worker.BlockStoreLocation;
+import tachyon.worker.block.BlockAccessEventListener;
+import tachyon.worker.block.BlockMetadataManager;
+import tachyon.worker.block.meta.BlockMeta;
+
+public class LFUEvictor implements Evictor, BlockAccessEventListener {
```

Comment:
I mean pure LFU like this implementation - blocks are incremented based on their total access counts. There are some issues like new blocks will be evicted very fast and old unused blocks (but used heavily a while ago) will not be evicted.